config/M64.config

```diff
@@ -37,6 +37,17 @@ process {
             }
         }
 
+    withName: filter_sSV_GRIDSS {
+        cpus = 1
+        memory = 20.GB
```

Comment:
Have you tested to see how much memory this step uses with a variety of samples.  Could there be an upper bound?   
Also, did you consider [this](https://github.com/PapenfussLab/gridss/blob/master/Readme.md):
`The Hartwig Medical Foundation has reimplemented gridss_somatic_filter in Java as [GRIPSS](https://github.com/hartwigmedical/hmftools/tree/master/gripss). GRIPSS is much faster, has additional features, and is the recommended tool for somatic filtering of GRIDSS output.`